src/index.ts

```diff
@@ -1,11 +1,14 @@
 import Video from './Video';
 
-export {default as FilterType} from './lib/FilterType';
-export {default as VideoResizeMode} from './lib/VideoResizeMode';
-export {default as TextTrackType} from './lib/TextTrackType';
-export {default as DRMType} from './lib/DRMType';
+export {default as ResizeMode} from 'react-native-video/src/types/ResizeMode';
```

Comment:
Should be relative path
```suggestion
export {default as ResizeMode} from ./types/ResizeMode';
```

Reply:
I found 2 more absolute path, I will remove it also